src/net.cpp

```diff
@@ -538,6 +539,14 @@ void CNode::SetAddrLocal(const CService& addrLocalIn) {
     }
 }
 
+Network CNode::ConnectedThroughNetwork() const
+{
+    if (IsInboundConn() && m_inbound_onion) {
+        return NET_ONION;
+    }
+    return addr.GetNetClass();
```

Comment:
optional style nit, could just simplify to a straight ternary
```diff
-    if (IsInboundConn() && m_inbound_onion) {
-        return NET_ONION;
-    }
-    return addr.GetNetClass();
+    return IsInboundConn() && m_inbound_onion ? NET_ONION : addr.GetNetClass();
```
or a one-line guard conditional
```diff
-    if (IsInboundConn() && m_inbound_onion) {
-        return NET_ONION;
-    }
+    if (IsInboundConn() && m_inbound_onion) return NET_ONION;
     return addr.GetNetClass();
```

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19998#issuecomment-703090033).